src/layout/layout.rs

```diff
@@ -1719,5 +1789,120 @@ mod tests {
             ]));
             assert_eq!([a.width, b.width, c.width, d.width], [0, 0, 0, 100]);
         }
+
+        #[test]
+        fn flex() {
```

Comment:
These test would probably be a good rstest test (there's a couple already in the repo for reference, but there's other methods that might suit this test.

Reply:
Changed it to rstest.